### ISSUE TITLE
android or engine: the app flags as successful a failed measurement

### ISSUE THREAD

**Issue by bassosimone** [ooni/probe#1744]
**Describe the bug**
A user reported to me that the Android app was saying that a measurement was successful when instead it was failed.

The key part of the measurements are these:

```JSON
{
  "network_events": null,
  "tls_handshakes": null,
  "dns_experiment_failure": "unknown_failure: lookup torproject.org: getaddrinfow: This is usually a temporary error during hostname resolution and means that the local server did not receive a response from an authoritative server.",
  "dns_consistency": "inconsistent",
  "control_failure": null,
  "tcp_connect": null,
  "requests": null,
  "http_experiment_failure": null, 
  "body_length_match": null,
  "body_proportion": 0,
  "status_code_match": null,
  "headers_match": null,
  "title_match": null,
  "accessible": null,
  "blocking": null
}
```

I suppose the issue could be related to the failure being an "unknown_failure". It may also be that this issue is not an issue of the app itself but rather an issue in the measurement engine, which provides the app with wrong summary keys.

For this reason, the title says "android or engine".

**To Reproduce**
Based on my understanding of the issue, it should suffice to test a domain that exists but for which the local resolver either returns NXDOMAIN or times out.

**Expected behavior**
The expected behavior in this case would have been to flag the measurement as an anomaly.

**Screenshots**
N/A

**System information (please complete the following information):**
 - Device: N/A
 - OS: Windows
 - Version: 3.5.0

**Additional context**
N/A

**Comment by bassosimone** [comment]
This issue will be fixed when we ship 3.11.0. The underlying problem is not correctly handling error codes on Windows, which is now fixed.

**Comment by bassosimone** [comment]
So, 3.11.0 is now 3.13.0 and it has been released!
